### PR TITLE
add boilerplate copied from ClusterLabs/ha_cluster_exporter

### PR DESCRIPTION
**Pull request by stefanotorresi**
The boilerplate refers to the name `sap-host-exporter` already, so wait until we settle that before merging.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -1,9 +1,69 @@
...
+}
+
+func init() {
+	config.SetConfigName("sap-host-exporter")
+	config.AddConfigPath("./")
+	config.AddConfigPath("$HOME/.config/")
+	config.AddConfigPath("/etc/")
+	config.AddConfigPath("/usr/etc/")
+
+	flag.String("port", "9664", "The port number to listen on for HTTP requests")
```
Comment (MalloZup) [inline main.go]: do we have reserved this port upstreaM?

Comment (stefanotorresi) [reply main.go]: Ahh no, you're right, it's the same from the other project! We shall reserve a new one and update this diff!

File: collector/default_collector.go [hunk 2]
```diff
@@ -0,0 +1,76 @@
+package collector
+
+import (
+	"os"
+
+	"github.com/pkg/errors"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const NAMESPACE = "sap"
```
Comment (MalloZup) [inline collector/default_collector.go]: I was thinking if perhaps sap or `sap_host` . we can find out later

Comment (stefanotorresi) [reply collector/default_collector.go]: Bear in mind that we will probably have a bunch of subsystems to group the metrics in, like `ascs`  and `ers`, this is why I left "host" out.

Comment (MalloZup) [reply collector/default_collector.go]: This is minor. I think we can resolve it later. No need right now

**Pull request merged**